### ISSUE TITLE
Agent status JSON output cannot be processed

### ISSUE THREAD

**Issue by j-vizcaino** [DataDog/datadog-agent#3047]
```
Agent 6.9.0 - Commit: 4bbd2c9 - Serialization version: 4.7.1
```

**Describe what happened:**

Running `agent status -j` or `agent status -p` prints informative message to `stdout`, making the output unparsable.

```bash
$ agent status -j
Getting the status from the agent.

{...}
```

**Describe what you expected:**

Messages for the end user should be sent to `stderr`.
This way, one would be able to run `agent status -j | jq ...` without hitting an error.

**Steps to reproduce the issue:**

Run `agent status -j | jq .`

**Comment by arbll** [comment]
Hi @j-vizcaino,

Thanks for the report. This will be fixed with 6.10: https://github.com/DataDog/datadog-agent/pull/3012/

**Comment by mfpierre** [comment]
Fixed by #3012
